tools/create_vulnerability_doc.py

```diff
@@ -88,9 +115,22 @@ def format_quantities(cves):
             for release, cves in releases.items():
                 # Dropdown for each release -- expand to show vulnerabilities
                 file.write(f"<details>\n<summary>{release} -- {format_quantities(cves)}</summary>\n\n")
+
+                # Url of release page and url of latest release (if this is not the latest release)
+                file.write(f"- Release URL: [{release}]({release_urls[release]})\n\n")
+                latest_release = latest_release_info[pkg]
+                if release == latest_release['name']:
+                    file.write("- This is the latest release.\n\n")
+                else:
+                    file.write(f"- Latest release: [{latest_release['name']}]({latest_release['url']})\n\n")
+
                 for cve in cves:
                     # Bullet point for each vulnerability in this release
                     file.write(f"- **({cve['severity']} severity) {cve['id']}**: {cve['details']}\n")
+                    # Show if this vulnerability is resolved in the latest release
+                    if (release != latest_release_info[pkg]['name'] and
+                            cve['id'] not in latest_release_info[pkg]['cve_ids']):
+                        file.write("  - **This vulnerability is resolved in the latest release.**\n")
```

Comment:
How about this format?

If it is the latest release, you can use the affected release URL above, and otherwise, it provides the latest release URL below every vulnerability and says whether the vuln has been resolved or not.

![newvulns](https://github.com/ZOSOpenTools/meta/assets/48895043/42b7838f-f887-451a-939f-00853415d79a)

Reply:
Love it!